src/pages/index.astro

```diff
@@ -4,15 +4,42 @@ import "../index.scss";
 ---
 
 <Layout title="Welcome to Astro.">
-  <main>
-<section class="hero container">
-  <div class="hero__content">
-    <h1 class="hero--title">Travel without limits!</h1>
-      <p class="hero--description">
-        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta ut facere aliquam impedit ullam minusreprehenderit quia in laudantium sint.</p>
-       <button class="hero--cta">Check offer</button>
-  </div>
-    </section>
-  </main>
+  <body>
```

Comment:
this body element is actually wrong

Reply:
take a look at `Layout` component in `src/layouts/Layout.astro`.
It has a `<slot />` element in which page content (like `pages/index.astro`) is being rendered

Reply:
Deleted